tripal_chado/src/TripalStorage/ChadoRecords.php

```diff
@@ -1123,10 +1123,16 @@ public function getColumnValue(string $base_table, string $table_alias, int $del
       return NULL;
     }
 
-    return $items[$delta]['values'][$column_alias];
+    // If the values were set then return it, otherwise return NULL;
+    if ($items[$delta]['values'] == TRUE) {
```

Comment:
I'm confused by this line. Did you mean to check has_values here instead?
```suggestion
    if ($items[$delta]['has_values'] === TRUE) {
```